experiment/experiment.go

```diff
@@ -142,7 +142,10 @@ func (e *Experiment) Measure(
 		return
 	}
 	measurement = e.newMeasurement(input)
+	start := time.Now()
 	err = e.DoMeasure(ctx, e.Session, &measurement, e.Callbacks)
+	stop := time.Now()
+	measurement.MeasurementRuntime = stop.Sub(start).Seconds()
```

Comment:
This is cool because:
> this code always computes a positive elapsed time, even if the wall clock is changed during the operation being timed

Reply:
Yes, a number of techniques are being implemented in `netx` and here. Also, this is a bug fix that I noticed when understanding the output of this experiment :).